src/dotnet/SlnProjectExtensions.cs

```diff
@@ -1,20 +1,17 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using Microsoft.DotNet.Cli.Sln.Internal;
 using System.Collections.Generic;
 using System.IO;
-using System.Linq;
 
 namespace Microsoft.DotNet.Tools.Common
 {
     internal static class SlnProjectExtensions
     {
-        public static IList<string> GetSolutionFoldersFromProject(this SlnProject project)
+        public static IList<string> GetSolutionFoldersFromProjectPath(string projectFilePath)
```

Comment:
I just noticed this change no longer makes this an extension method.

Given that we only have one caller of this in `AddProjectToSolutionCommand `, perhaps we should just move these few methods into that class and delete this file?

Reply:
I was unsure if I should do this, but since you think the same .. :)